R/PCA.R

```diff
@@ -615,9 +615,12 @@ getShapeScale <- function(plotData, shape, shapePalette, colourScaleType = NULL,
 
 getGeomPoint <- function(cV, shape, my_scale_shape, pointSize, alpha) {
 
-  filledShapes <- ifelse(my_scale_shape$scale_name == "manual" & any(my_scale_shape$palette(1) %in% 21:25),
-                         TRUE, FALSE)
-
+  if(any(my_scale_shape$palette(1) %in% 21:25)){
+    filledShapes <- TRUE
+  } else {
+    filledShapes <- FALSE 
+  }
+  
```

Comment:
@Steven-M-Hill , with the most recent version of ggplot2, `scale_name` doesn't appear to be an option of my_scale_shape, and this was evaluating to NULL.

Reply:
I'll take a look

Reply:
I think this is fine for now. `my_scale_shape` is either a manual scale or, if no shape aesthetic is used, identity scale. At the moment the latter sets the shape to 16 and is hard-coded.
In the future we may want to add in the option to change the default shape when there is no shape aesthetic. The code here would need to be changed at this point.

Reply:
Sure, we can always change things later on.